cmd/validate/image.go

```diff
@@ -363,6 +365,9 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 	cmd.Flags().StringVarP(&data.rekorURL, "rekor-url", "r", data.rekorURL,
 		"Rekor URL. Overrides rekorURL from EnterpriseContractPolicy")
 
+	cmd.Flags().BoolVar(&data.ignoreRekor, "ignore-rekor", data.ignoreRekor,
+		"Skip Rekor checks during validation.")
```

Comment:
Maybe..?

```suggestion
		"Skip Rekor transparency log checks during validation.")
```

Reply:
Good idea. Also added to the task param.